staking/contracts/contracts.py

```diff
@@ -241,6 +243,9 @@ def init(
     """Initialise the newly deployed contract, funding it with a minimum
     balance and allowing it to opt in to the request assets."""
     return Seq(
```

Comment:
Should the `init` method be able to be called multiple times? It feels like there should be some sort of idempotence enforced here, but I'm wondering if an app should be able to initialize multiple assets.